docs/topics/http/file-uploads.txt

```diff
@@ -122,6 +122,54 @@ field in the model::
             form = UploadFileForm()
         return render(request, 'upload.html', {'form': form})
 
+Uploading multiple files
+------------------------
+
+If you want to upload multiple files at the same time, you need to set the
+``multiple`` boolean attribute of the input element in your
+:class:`~django.forms.ModelForm` first:
+
+.. snippet::
+    :filename: forms.py
+
+    from django import forms
+    from .models import ModelWithFileField
+
+    class FileFieldModelForm(forms.ModelForm):
+
+        file_field = forms.FileField(widget=forms.ClearableFileInput(attrs={'multiple': True}))
```

Comment:
I mean, when you access a FileField on a model, you are given an instance of FieldFile as a proxy for accessing the underlying file. This can't handle multiple files, can it?